### PR TITLE
OCPBUGS-19943: Fix hybrid-overlay parameters required for certificate management

### PR DESCRIPTION
**Pull request by kyrtapz**
First commit bumps ovn-kubernetes submodule.
Second commit adjusts hybrid-overlay parameters required to use a per-node certificate.

### CODE REVIEW

**Review by sebsoto — APPROVED** [review]
Thanks for the quick fix, LGTM

**Comment by sebsoto** [comment]
This is the fix that is being brought in: https://github.com/openshift/ovn-kubernetes/pull/1919/commits/ac329f818509ca282a5791dd4a4751c1cf2bba84

**Comment by sebsoto** [comment]
/retitle OCPBUGS-19943: Fix hybrid-overlay parameters required for certificate management

File: pkg/services/services.go [hunk 1]
```diff
@@ -102,8 +102,8 @@ func azureCloudNodeManagerConfiguration() servicescm.Service {
 
 // hybridOverlayConfiguration returns the Service definition for hybrid-overlay
 func hybridOverlayConfiguration(vxlanPort string, debug bool) servicescm.Service {
-	hybridOverlayServiceCmd := fmt.Sprintf("%s --node NODE_NAME --k8s-kubeconfig %s --windows-service "+
-		"--logfile "+"%s\\hybrid-overlay.log", windows.HybridOverlayPath, windows.KubeconfigPath,
+	hybridOverlayServiceCmd := fmt.Sprintf("%s --node NODE_NAME --bootstrap-kubeconfig=%s --cert-dir=%s --cert-duration=24h "+
```
Comment (jrvaldes) [inline pkg/services/services.go:105]: @kyrtapz  `--cert-duration=24h` means the cert will renew or expire in 24 h? 

Can the `hybrid overlay` service modify annotations after this duration?

**Review by jrvaldes — COMMENTED** [review]
@kyrtapz  thanks for working on this. LGTM.

File: pkg/services/services.go (hunk 1 above)
Comment (kyrtapz) [reply pkg/services/services.go:105]: This means that the certificate will be valid for 24h, before it expires the hybrid-overlay binary will perform automatic rotation so there should be no issues there.

Comment (jrvaldes) [reply pkg/services/services.go:105]: Thanks, went back to the [thread](https://redhat-internal.slack.com/archives/CDCP2LA9L/p1695977610275759) and found the answer. 😄

Comment (aravindhp) [reply pkg/services/services.go:105]: How do the certs end up in the CNI conf directory?

Comment (aravindhp) [reply pkg/services/services.go:105]: Seems like the hybrid-overlay will created them.

Comment (kyrtapz) [reply pkg/services/services.go:105]: The certificate files are created by hybrid-overlay based on a CSR request that gets approved.

**Pull request merged**